disco_aws_automation/disco_vpc.py

```diff
@@ -227,6 +229,33 @@ def _open_customer_ports(self):
                 src_security_group_group_id=self.networks["dmz"].security_group.id
             )
 
+    def _update_nat_gateways(self, network):
+        eips = self.get_config("{0}_nat_gateways".format(network.name))
```

Comment:
What if I want to create a sandbox env with NAT? (First question, is do we want to do this.)

In such a case I would need to allocate and EIPs for NAT and assign them manually through config. I think it would be handy to use dynamic public IPs and auto add them to route.